docs/aad-application-configuration.md

```diff
@@ -50,20 +50,25 @@ Snippet Output :
 
 > Note : Don't use a hard coded value like `OSA_AAD_SECRET=MyAw3s0meP@ssw0rd!` in production. You may want to generate a random secret and copy the value in a safe place.
 
+Put the value of the output's `appId` in the variable `OSA_AAD_ID`, following the example above:
+
+```azurecli-interactive
+OSA_AAD_ID=57b4f673-af45-1223-1234-efb12fc0cd16
```

Comment:
What about to keep the shell script to get the app ID dynamically? 
`az ad app show --id $OSA_AAD_REPLY_URL --query appId -o tsv`

Reply:
you're using OSA_AAD_REPLY_URL in your suggestion but that may be wrong? if there's another way to get the appId from an `az ad app show` call that doesn't require the appId as a paremeter, that would be fine I think.

Reply:
I am using it because we are using it to create the app at the first time, but agreed this could be confusing.

We can create a new `var` like `$OSA_AAD_NAME`
Then `az ad app create --display-name $OSA_CLUSTER_NAME --key-type Password --password $OSA_AAD_SECRET --identifier-uris $OSA_AAD_NAME --reply-urls $OSA_AAD_REPLY_URL` to create the app.

Finally : `az ad app show --id $OSA_AAD_NAME --query appId -o tsv`